cfe_internal/enterprise/federation/federation.cf

```diff
@@ -410,6 +410,27 @@ body contain contain_transport_user
     useshell => "true";
 }
 
+bundle monitor measure_psql
```

Comment:
So this monitor bundle is in the federation namespace? Ok, I haven't ever namespaced a measurement before, glad to know it works in some way.

Reply:
not sure.